samples/ExportHistoryWebApp/ExportJobController.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.ExportHistory;
+using ExportHistoryWebApp.Models;
+
+namespace ExportHistoryWebApp.Controllers;
+
+/// <summary>
+/// Controller for managing export history jobs through a REST API.
+/// Provides endpoints for creating, reading, listing, and deleting export jobs.
+/// </summary>
+[ApiController]
+[Route("export-jobs")]
+public class ExportJobController : ControllerBase
+{
+    readonly ExportHistoryClient exportHistoryClient;
+    readonly ILogger<ExportJobController> logger;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ExportJobController"/> class.
+    /// </summary>
+    /// <param name="exportHistoryClient">Client for managing export history jobs.</param>
+    /// <param name="logger">Logger for recording controller operations.</param>
+    public ExportJobController(
+        ExportHistoryClient exportHistoryClient,
+        ILogger<ExportJobController> logger)
+    {
+        this.exportHistoryClient = exportHistoryClient ?? throw new ArgumentNullException(nameof(exportHistoryClient));
+        this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+    }
+
+    /// <summary>
+    /// Creates a new export job based on the provided configuration.
+    /// </summary>
+    /// <param name="request">The export job creation request.</param>
+    /// <returns>The created export job description.</returns>
+    [HttpPost]
+    public async Task<ActionResult<ExportJobDescription>> CreateExportJob([FromBody] CreateExportJobRequest request)
```

Comment:
Instead of having users create their own request, should we just show them using the options here as the domain object? I want to make sure we set the correct precedent for them.

Reply:
ditto